README.md

```diff
@@ -85,6 +91,26 @@ parameters:
 ```
 For NFS, you can also specify other options, as described in [exports(5)][man exports].
 
+The following example configures a storage class using the `Auto` type. The provisioner
+will decide whether [HostPath][hostpath] or [NFS][nfs] will be used based on where the
+pods are being scheduled.
```

Comment:
Maybe I misunderstood something. Didn't we recently discuss the side-effect of scheduling, and therefore decide based on the requested Access mode instead of the scheduled node?

Reply:
Do you mean we should set node affinity in the Auto mode when the volume claim contains ReadWriteOnce instead of letting the scheduler choose where to place the pods ? That's not what's currently happening in the current state of this PR but that's something we can do I suppose.

Reply:
Yes, exactly.

The storage class contains the ZFS node name. If the requested Volume using this class is only RWO or RWOP, then we let the pod to be run on that node using HostPath (the volume contains the affinity rule).

However, if the volume contains RWX or ROX, then we'll let the scheduler decide, and set the volume to NFS. That way, it _could_ happen that if pods are scheduled on the same node as the ZFS node, it would still be mounted as NFS. But it also avoids the side effect, as all pods will be mounted as NFS regardless where they get scheduled